HHS/Head-Start-TTADP
# [TTAHUB-1584] - Fixed filter pills breaking at smaller device widths

## Description of change

- Adjusted FilterPills.css to keep pills from breaking, tweaks to alignment, and responsive max widths.
- Created one flex gap utility class in App.scss (more can be added if needed)
- Used new flex gap utility class across pages using filter to help layout filter pills evenly as they wrap


## How to test
Change window/device width or zoom in to see how filter pills wrap (filter name + condition text should be closer to pill when it wraps, than it is to the next pill group, to denote relationship). Max-width of pill text (value) changes to 200px at < 640px device width.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1584

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
